ogr/services/gitlab.py

```diff
@@ -341,10 +353,22 @@ def pr_merge(self, pr_id: int) -> "PullRequest":
         return self._pr_from_gitlab_object(pr)
 
     def get_pr_labels(self, pr_id: int) -> List:
```

Comment:
Maybe a more specific type?

```suggestion
    def get_pr_labels(self, pr_id: int) -> List[str]:
```